awscli/customizations/s3/filegenerator.py

```diff
@@ -116,7 +116,7 @@ class FileGenerator(object):
     ``FileInfo`` objects to send to a ``Comparator`` or ``S3Handler``.
     """
     def __init__(self, client, operation_name, follow_symlinks=True,
-                 page_size=None, result_queue=None):
+                 page_size=None, result_queue=None, request_parameters=None):
```

Comment:
Does it make sense to move page_size in the request_parameters dict with an entry for `ListObjects`?